js/game.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-named-as-default-member */
+/* eslint-disable import/no-named-as-default */
+import Scene1 from './scene1.js';
+import Scene2 from './scene2.js';
+import Scene3 from './scene3.js';
+import Board from './board.js';
+import Player from './player.js';
+
+const Game = (() => {
+  const config = {
+    // eslint-disable-next-line no-undef
+    type: Phaser.AUTO,
+    width: window.innerWidth,
+    height: window.innerHeight,
+    parent: true,
+    dom: {
+      createContainer: true,
+    },
+    scene: [Scene1, Scene2, Scene3],
+  };
+
+  const status = (() => {
+    const win = () => 'win';
+    const playing = () => 'playing';
+    const draw = () => 'draw';
+
+    return { win, playing, draw };
+  })();
+
+  // lb = logic board
+  const inspectStatus = (lb) => {
+    const lastPos = lb.length - 1;
+
+    for (let i = 0; i <= lastPos; i += 1) {
```

Comment:
**[OPTIONAL]**

Instead of using a `for` loop here, we can (and actually should) use `forEach`, `for of`.